lib/please_cli/please_cli/terraform_route53_config.py

```diff
@@ -0,0 +1,135 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+import click
+
+import please_cli.config
+
+HEADER = '''
```

Comment:
I think we should move these generated terraform configs to a separate file than rds.tf.  rds.tf should only contain the contents of the HEADER except for the root-alias resource since that is related to the records that are being generated.

Reply:
Idea was that everything under ``mozilla-releng.net`` domain would sooner or later be managed by this repository. Currently only [coalesce](coalesce.mozilla-releng.net) service is being developed outside. Since you are a maintainer of it, would you find it useful to move code under ``mozilla-releng/services``?
Pros: Same conventions that are used by all our services would apply also to coalesce service. The biggest advantage I see would be that tooling in ``mozilla-releng/services`` is keeping every dependency up-to-date and that monitoring is managed by multiple people.
Cons: We are touching an existing&working service, maybe it is better to not touch it at all.

I don't have strong opinion on this, by maybe this would be releasing you of some work, if you want we can also chat about this if you have more questions about Pros/Cons.

I'm ok to remove non-``mozilla-releng/services`` DNS stuff from the generated output if you find this to be a more clean approach (I think many times I try to do too much at once :P).

Reply:
I get the idea, I'm just playing devil's advocate.  I'd even be open to the idea of not using tf for these apps but we should really work out the pros/cons there too.  For now, let's move forward on this.  It is a solution and shouldn't bit rot while you are out.  It does need some fixes since I noticed the output generate had a conflict and resource naming didn't make sense.  It can be fixed on your return though. 👍

And thank you opening bugs per app.  I prefer to iterate over small things than lump all things together.